.circleci/config.yml

```diff
@@ -140,9 +157,9 @@ commands:
       - restore_cache:
           name: "restore e2e cache"
           keys:
-            - v1-e2e-cache-{{ .Branch }}-{{ checksum "~/workbench/e2e/yarn.lock" }}
-            - v1-e2e-cache-master
-            - v1-e2e-cache-
+            - v1-e2e-dependency-cache-{{ .Branch }}-{{ checksum "~/workbench/e2e/yarn.lock" }}
```

Comment:
nit: this feels implicit to me - what else would we be caching? I don't think the change is needed

Reply:
reverted. thought it would be good to match command name `install_e2e_dependencies`.